bom-internal/pom.xml

```diff
@@ -76,13 +76,18 @@ POM as their parent.
         <skipTests>false</skipTests>
         <skipITs>true</skipITs>
     </properties>
-
+    <!-- This section is identical to the root pom.xml -->
```

Comment:
Possible comment improvement, this reads like an observation "I noticed that this section is the same as the root pom.xml"
Did you mean something like  "This section needs to be the same as the root pom.xml for this work"?